lib/node_modules/@stdlib/random/base/mt19937/lib/factory.js

```diff
@@ -88,21 +101,53 @@ function factory( seed ) {
 		mt[ mti ] >>>= 0;
 	}
 
+	if ( seedIsArray ) {
+		for (; k > 0; k--) {
+			s = mt[ i - 1 ] ^ ( mt[ i - 1 ] >>> 30 );
+
+			mt[i] = (mt[i] ^ (((((s & 0xffff0000) >>> 16) * 1664525) << 16) +
+				( (s & 0x0000ffff) * 1664525 ) ) ) + seed[ j ] + j;
+			mt[i] >>>= 0;
+			i += 1;
+			j += 1;
+			if (i >= N) {
+				mt[0] = mt[N - 1];
+				i = 1;
+			}
+			if (j >= seed.length) {
+				j = 0;
+			}
+		}
+
+		for (k = N - 1; k; k--) {
+			s = mt[i - 1] ^ (mt[i - 1] >>> 30);
+			mt[i] = (mt[i] ^ (((((s & 0xffff0000) >>> 16) * 1566083941) << 16) +
```

Comment:
Our preference is not to split lines. Instead, we tend to disable the lint rule. 

```
// eslint-disable-line max-len
```